### DISCUSSION TITLE
I found that lanraragi can't scan all the comics I have stored

### DISCUSSION THREAD

**Discussion by nyanpasunyan** [Difegue/LANraragi#679] [Q&A]
I am using the windows version. I found that lanraragi can't scan all the comics I have stored. I have about 10,000 comics, all in zip format, but every time lanraragi only scans more than 1,000 comics, it stops scanning and has to restart before it starts scanning again. And it stops scanning when it scans a few hundred copies.

**Comment by Difegue** [comment]
👋 Sounds like #282.  
Since you're on windows, you might try switching to WSL2 in the app settings and see if that improves things. 👍

**Comment by nyanpasunyan** [comment]
> 👋 Sounds like #282. Since you're on windows, you might try switching to WSL2 in the app settings and see if that improves things. 👍



> 👋 Sounds like #282. Since you're on windows, you might try switching to WSL2 in the app settings and see if that improves things. 👍

I changed to wsl2, but still no improvement.

**Comment by Difegue** [comment]
Sorry for the late reply, this issue kinda slipped through the cracks 🥲  

If WSL isn't the cause of the issue it might be caused by the folder you're storing your zips in -- Is it a network drive?  
Sadly filesystem performance is usually hit-or-miss with netdrives and that's an issue I can't easily fix.

**Reply by Difegue** [reply]
Moving to a discussion -- If there are any more occurences of this please put them in #282! Thanks 👍

**Comment by skyrim663** [comment]
Other than moving files to the content directory in small batches, is there any way to get LLR to add archives correctly? I've tried reinstalling and scanning and it doesn't seem to work, after adding 1000~2000 files it gets stuck and doesn't log any further errors!

**Reply by Kaga990509** [reply]
I had meet the same situation this days，now i found it is beacause some filename is too long. It is true longth in windows system but it is too long for linux system.

**Comment by hucurve** [comment]
I had this problem too, and Lanraghi was perfect for me except for this pain point,and i cant figure out what to do
